codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/codegen/SmithyGoDependency.java

```diff
@@ -95,7 +95,7 @@ public final class SmithyGoDependency {
   public static final GoDependency GO_JMESPATH = goJmespath(null);
   public static final GoDependency MATH = stdlib("math");
 
-  private static final String SMITHY_SOURCE_PATH = "github.com/aws/smithy-go";
+  public static final String SMITHY_SOURCE_PATH = "github.com/aws/smithy-go";
```

Comment:
We have used public static variable in a lot of places and smithy dafny will not be in production code. So, I made it public instead of having a getter method.